docs/assembly.rst

```diff
@@ -333,11 +333,12 @@ push their entry label (with virtual function resolution applied). The calling s
 in solidity are:
 
  - the caller pushes return label, arg1, arg2, ..., argn
```

Comment:
I think the point is the call consumes these elements, not that the caller pushes these elements.  In the end, when we calculate `m - n - 1`, we calculate what the call does, no what the caller does.